modules/jetstream-maxtext-deployment/README.md

```diff
@@ -53,11 +55,21 @@ cat ./templates/deployment.yaml.tftpl >> "$JETSTREAM_MANIFEST"
 PODMONITORING_MANIFEST=$(mktemp)
 cat ./templates/podmonitoring.yaml.tftpl >> "$PODMONITORING_MANIFEST"
 
-if [ "$METRICS_PORT" != "" ]; then
-    cat $PODMONITORING_MANIFEST | sed "s/\${metrics_port}/$METRICS_PORT/g" >> "$PODMONITORING_MANIFEST"
+PODMONITORING_TPU_MANIFEST=$(mktemp)
+cat ./templates/podmonitoring-tpu.yaml.tftpl >> "$PODMONITORING_TPU_MANIFEST"
+
+if [ "$METRICS_PORT" != "" ] && [ "$METRICS_SCRAPE_INTERVAL" != "" ]; then
```

Comment:
Should this if statement be updated since METRICS_SCRAPE_INTERVAL is no longer defined?